dask_expr/_util.py

```diff
@@ -0,0 +1,10 @@
+def _convert_to_list(column):
```

Comment:
```suggestion
def _convert_to_list(column) -> list | None:
```

Also, do we want a name  (`_columns_as_list`) and description (?) clarifying that the purpose of this utility is to normalize a column/columns  argument to `list`?  Or, do we want to keep it more general? I'm on the fence about this.

Reply:
I generally prefer the ``_convert_to_foo`` pattern. We don't really have column-specific logic in there.